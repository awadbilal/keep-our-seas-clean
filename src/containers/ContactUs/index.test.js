import React from "react";
import renderer from "react-test-renderer";
import ContactUs from "./index";

it("renders correctly", () => {
  const tree = renderer.create(<ContactUs />).toJSON();
  expect(tree).toMatchSnapshot();
});
