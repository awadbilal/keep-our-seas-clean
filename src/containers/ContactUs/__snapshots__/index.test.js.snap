// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="main pt-5 pb-5 fdb-block container-fluid"
  >
    <div
      className="row"
    >
      <div
        className="col-md-8 offset-md-2 text-center col"
      >
        <h1
          className="title"
        >
          contact.header.title
        </h1>
      </div>
    </div>
  </div>
  <div
    className="pt-5 pb-5 fdb-block container-fluid"
  >
    <div
      className="align-items-center row"
    >
      <div
        className="px-4 col-lg-3 col-md-12 col-sm-12 offset-lg-1"
      >
        <h2
          className="title"
        >
          contact.team.title
        </h2>
        <p
          className="mt-4"
        >
          contact.team.paragraph
        </p>
      </div>
      <div
        className="px-4 col-lg-7 col-md-12 col-sm-12"
      >
        <div
          className="row row-cols-md-12 row-cols-sm-12"
        >
          <div
            className="col-md-6 col-sm-12"
          >
            <div
              className="align-items-center pt-4 row row-cols-md-12 row-cols-sm-12"
            >
              <div
                className="team-section-picture col-lg-4 col-md-6 col-sm-1"
              >
                <img
                  alt="Moumen Hallak"
                  className="memberImg"
                  src="./images/moumen.jpg"
                />
              </div>
              <div
                className="team-section-content col-lg-8 col-md-6 col-sm-4"
              >
                <h4>
                  Moumen Hallak
                </h4>
                <a
                  href="https://www.linkedin.com/in/moumen-hallak/"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <svg
                    className="contactIcons p-1"
                    fill="#383838"
                    height={30}
                    style={
                      Object {
                        "cursor": "pointer",
                      }
                    }
                    viewBox="0 0 16 16"
                    width={30}
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 1.146C0 .513.526 0 1.175 0h13.65C15.474 0 16 .513 16 1.146v13.708c0 .633-.526 1.146-1.175 1.146H1.175C.526 16 0 15.487 0 14.854V1.146zm4.943 12.248V6.169H2.542v7.225h2.401zm-1.2-8.212c.837 0 1.358-.554 1.358-1.248-.015-.709-.52-1.248-1.342-1.248-.822 0-1.359.54-1.359 1.248 0 .694.521 1.248 1.327 1.248h.016zm4.908 8.212V9.359c0-.216.016-.432.08-.586.173-.431.568-.878 1.232-.878.869 0 1.216.662 1.216 1.634v3.865h2.401V9.25c0-2.22-1.184-3.252-2.764-3.252-1.274 0-1.845.7-2.165 1.193v.025h-.016a5.54 5.54 0 0 1 .016-.025V6.169h-2.4c.03.678 0 7.225 0 7.225h2.4z"
                    />
                  </svg>
                </a>
                <a
                  href="mailto:moumen95@gmail.com"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <svg
                    className="contactIcons p-1"
                    fill="#383838"
                    height={34}
                    style={
                      Object {
                        "cursor": "pointer",
                      }
                    }
                    viewBox="0 0 16 16"
                    width={34}
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M.05 3.555A2 2 0 0 1 2 2h12a2 2 0 0 1 1.95 1.555L8 8.414.05 3.555zM0 4.697v7.104l5.803-3.558L0 4.697zM6.761 8.83l-6.57 4.027A2 2 0 0 0 2 14h12a2 2 0 0 0 1.808-1.144l-6.57-4.027L8 9.586l-1.239-.757zm3.436-.586L16 11.801V4.697l-5.803 3.546z"
                    />
                  </svg>
                </a>
              </div>
            </div>
          </div>
          <div
            className="col-md-6 col-sm-12"
          >
            <div
              className="align-items-center pt-4 row row-cols-md-12 row-cols-sm-12"
            >
              <div
                className="team-section-picture col-lg-4 col-md-6 col-sm-1"
              >
                <img
                  alt="Bilal Avvad"
                  className="memberImg"
                  src="./images/bilal.png"
                />
              </div>
              <div
                className="team-section-content col-lg-8 col-md-6 col-sm-4"
              >
                <h4>
                  Bilal Avvad
                </h4>
                <a
                  href="https://www.linkedin.com/in/bilal-avvad/"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <svg
                    className="contactIcons p-1"
                    fill="#383838"
                    height={30}
                    style={
                      Object {
                        "cursor": "pointer",
                      }
                    }
                    viewBox="0 0 16 16"
                    width={30}
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 1.146C0 .513.526 0 1.175 0h13.65C15.474 0 16 .513 16 1.146v13.708c0 .633-.526 1.146-1.175 1.146H1.175C.526 16 0 15.487 0 14.854V1.146zm4.943 12.248V6.169H2.542v7.225h2.401zm-1.2-8.212c.837 0 1.358-.554 1.358-1.248-.015-.709-.52-1.248-1.342-1.248-.822 0-1.359.54-1.359 1.248 0 .694.521 1.248 1.327 1.248h.016zm4.908 8.212V9.359c0-.216.016-.432.08-.586.173-.431.568-.878 1.232-.878.869 0 1.216.662 1.216 1.634v3.865h2.401V9.25c0-2.22-1.184-3.252-2.764-3.252-1.274 0-1.845.7-2.165 1.193v.025h-.016a5.54 5.54 0 0 1 .016-.025V6.169h-2.4c.03.678 0 7.225 0 7.225h2.4z"
                    />
                  </svg>
                </a>
                <a
                  href="mailto:awadbilal99@gmail.com"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <svg
                    className="contactIcons p-1"
                    fill="#383838"
                    height={34}
                    style={
                      Object {
                        "cursor": "pointer",
                      }
                    }
                    viewBox="0 0 16 16"
                    width={34}
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M.05 3.555A2 2 0 0 1 2 2h12a2 2 0 0 1 1.95 1.555L8 8.414.05 3.555zM0 4.697v7.104l5.803-3.558L0 4.697zM6.761 8.83l-6.57 4.027A2 2 0 0 0 2 14h12a2 2 0 0 0 1.808-1.144l-6.57-4.027L8 9.586l-1.239-.757zm3.436-.586L16 11.801V4.697l-5.803 3.546z"
                    />
                  </svg>
                </a>
              </div>
            </div>
          </div>
          <div
            className="col-md-6 col-sm-12"
          >
            <div
              className="align-items-center pt-4 row row-cols-md-12 row-cols-sm-12"
            >
              <div
                className="team-section-picture col-lg-4 col-md-6 col-sm-1"
              >
                <img
                  alt="Isa Tekinkaya"
                  className="memberImg"
                  src="./images/avatar.png"
                />
              </div>
              <div
                className="team-section-content col-lg-8 col-md-6 col-sm-4"
              >
                <h4>
                  Isa Tekinkaya
                </h4>
                <a
                  href="https://www.linkedin.com/in/isa-tekinkaya-7020131a7/"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <svg
                    className="contactIcons p-1"
                    fill="#383838"
                    height={30}
                    style={
                      Object {
                        "cursor": "pointer",
                      }
                    }
                    viewBox="0 0 16 16"
                    width={30}
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 1.146C0 .513.526 0 1.175 0h13.65C15.474 0 16 .513 16 1.146v13.708c0 .633-.526 1.146-1.175 1.146H1.175C.526 16 0 15.487 0 14.854V1.146zm4.943 12.248V6.169H2.542v7.225h2.401zm-1.2-8.212c.837 0 1.358-.554 1.358-1.248-.015-.709-.52-1.248-1.342-1.248-.822 0-1.359.54-1.359 1.248 0 .694.521 1.248 1.327 1.248h.016zm4.908 8.212V9.359c0-.216.016-.432.08-.586.173-.431.568-.878 1.232-.878.869 0 1.216.662 1.216 1.634v3.865h2.401V9.25c0-2.22-1.184-3.252-2.764-3.252-1.274 0-1.845.7-2.165 1.193v.025h-.016a5.54 5.54 0 0 1 .016-.025V6.169h-2.4c.03.678 0 7.225 0 7.225h2.4z"
                    />
                  </svg>
                </a>
                <a
                  href="mailto:Isatekinkaya@yahoo.de"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <svg
                    className="contactIcons p-1"
                    fill="#383838"
                    height={34}
                    style={
                      Object {
                        "cursor": "pointer",
                      }
                    }
                    viewBox="0 0 16 16"
                    width={34}
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M.05 3.555A2 2 0 0 1 2 2h12a2 2 0 0 1 1.95 1.555L8 8.414.05 3.555zM0 4.697v7.104l5.803-3.558L0 4.697zM6.761 8.83l-6.57 4.027A2 2 0 0 0 2 14h12a2 2 0 0 0 1.808-1.144l-6.57-4.027L8 9.586l-1.239-.757zm3.436-.586L16 11.801V4.697l-5.803 3.546z"
                    />
                  </svg>
                </a>
              </div>
            </div>
          </div>
          <div
            className="col-md-6 col-sm-12"
          >
            <div
              className="align-items-center pt-4 row row-cols-md-12 row-cols-sm-12"
            >
              <div
                className="team-section-picture col-lg-4 col-md-6 col-sm-1"
              >
                <img
                  alt="Yeşim Nur Akar"
                  className="memberImg"
                  src="./images/yesim.jpg"
                />
              </div>
              <div
                className="team-section-content col-lg-8 col-md-6 col-sm-4"
              >
                <h4>
                  Yeşim Nur Akar
                </h4>
                <a
                  href="https://www.linkedin.com/in/yesimnurakar/"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <svg
                    className="contactIcons p-1"
                    fill="#383838"
                    height={30}
                    style={
                      Object {
                        "cursor": "pointer",
                      }
                    }
                    viewBox="0 0 16 16"
                    width={30}
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M0 1.146C0 .513.526 0 1.175 0h13.65C15.474 0 16 .513 16 1.146v13.708c0 .633-.526 1.146-1.175 1.146H1.175C.526 16 0 15.487 0 14.854V1.146zm4.943 12.248V6.169H2.542v7.225h2.401zm-1.2-8.212c.837 0 1.358-.554 1.358-1.248-.015-.709-.52-1.248-1.342-1.248-.822 0-1.359.54-1.359 1.248 0 .694.521 1.248 1.327 1.248h.016zm4.908 8.212V9.359c0-.216.016-.432.08-.586.173-.431.568-.878 1.232-.878.869 0 1.216.662 1.216 1.634v3.865h2.401V9.25c0-2.22-1.184-3.252-2.764-3.252-1.274 0-1.845.7-2.165 1.193v.025h-.016a5.54 5.54 0 0 1 .016-.025V6.169h-2.4c.03.678 0 7.225 0 7.225h2.4z"
                    />
                  </svg>
                </a>
                <a
                  href="mailto:yesimnurakar@gmail.com"
                  rel="noopener noreferrer"
                  target="_blank"
                >
                  <svg
                    className="contactIcons p-1"
                    fill="#383838"
                    height={34}
                    style={
                      Object {
                        "cursor": "pointer",
                      }
                    }
                    viewBox="0 0 16 16"
                    width={34}
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M.05 3.555A2 2 0 0 1 2 2h12a2 2 0 0 1 1.95 1.555L8 8.414.05 3.555zM0 4.697v7.104l5.803-3.558L0 4.697zM6.761 8.83l-6.57 4.027A2 2 0 0 0 2 14h12a2 2 0 0 0 1.808-1.144l-6.57-4.027L8 9.586l-1.239-.757zm3.436-.586L16 11.801V4.697l-5.803 3.546z"
                    />
                  </svg>
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="main pt-5 pb-5 fdb-block container-fluid"
  >
    <div
      className="align-items-center row"
    >
      <div
        className="px-4 col-lg-3 col-md-12 col-sm-12 offset-lg-1"
      >
        <h2
          className="title"
        >
          contact.getInTouch.title
        </h2>
        <p
          className="mt-4"
        >
          contact.getInTouch.paragraph
        </p>
      </div>
      <div
        className="px-4 col-lg-7 col-md-12 col-sm-12"
      >
        <div
          className="row row-cols-md-12 row-cols-sm-12"
        >
          <form
            className=""
          >
            <div
              className="mb-3 row"
            >
              <div
                className="col"
              >
                <label
                  className="form-label"
                  htmlFor="formGridName"
                >
                  form.name
                </label>
                <input
                  className="form-control"
                  id="formGridName"
                  name="name"
                  onChange={[Function]}
                  placeholder="formPlaceholder.name"
                  required={true}
                  type="text"
                  value=""
                />
              </div>
              <div
                className="col"
              >
                <label
                  className="form-label"
                  htmlFor="formGridSurname"
                >
                  form.surname
                </label>
                <input
                  className="form-control"
                  id="formGridSurname"
                  name="surname"
                  onChange={[Function]}
                  placeholder="formPlaceholder.surname"
                  required={true}
                  type="text"
                  value=""
                />
              </div>
            </div>
            <div
              className="mb-3"
            >
              <label
                className="form-label"
                htmlFor="formGridEmail"
              >
                form.email
              </label>
              <input
                className="form-control"
                id="formGridEmail"
                name="email"
                onChange={[Function]}
                placeholder="formPlaceholder.email"
                required={true}
                type="email"
                value=""
              />
            </div>
            <div
              className="mb-3"
            >
              <label
                className="form-label"
                htmlFor="exampleForm.ControlTextarea1"
              >
                form.message
              </label>
              <textarea
                className="form-control"
                id="exampleForm.ControlTextarea1"
                name="message"
                onChange={[Function]}
                placeholder="formPlaceholder.message"
                required={true}
                rows={3}
                value=""
              />
            </div>
            <button
              className="submitButton btn btn-primary"
              disabled={false}
              onClick={[Function]}
              type="submit"
            >
              form.submit
            </button>
          </form>
        </div>
      </div>
    </div>
  </div>
</div>
`;
